test/dist_optimizer_test.py

```diff
@@ -2,13 +2,14 @@
 
 import unittest
 
-from dist_utils import INIT_METHOD_TEMPLATE, dist_init
+from dist_utils import dist_init
 from torch import optim
 from torch.distributed.optim import DistributedOptimizer
 import torch
 import torch.distributed.autograd as dist_autograd
 import torch.distributed.rpc as rpc
 import threading
+from rpc_agent_test_fixture import RpcAgentTestFixture
```

Comment:
move from up to the front of import?

Reply:
Let me do a whole formatting in a separate PR.

Reply:
sure